README.md

```diff
@@ -1,11 +1,21 @@
 # Motoman
 
-[ROS-Industrial][] motoman meta-package.  See the [ROS wiki][] page for more information.  
+[![Build Status](http://build.ros.org/job/Idev__motoman__ubuntu_trusty_amd64/badge/icon)](http://build.ros.org/job/Idev__motoman__ubuntu_trusty_amd64)
+
+[ROS-Industrial][] Motoman metapackage. See the [ROS wiki][] page for more information.
+
+The [motoman_experimental][] repository contains additional packages.
+
 
 ## Contents
 
-This repo holds source code for all versions > groovy. For those versions <= groovy see: [SVN repo][]
+Branch naming follows the ROS distribution they are compatible with. `-devel`
+branches may be unstable. Releases are made from the distribution branches
+(`hydro`, `indigo`).
+
+Older releases may be found in the old ROS-Industrial [subversion repository][].
 
-[ROS-Industrial]: http://www.ros.org/wiki/Industrial
-[ROS wiki]: http://ros.org/wiki/motoman
-[SVN repo]: https://code.google.com/p/swri-ros-pkg/source/browse
+[ROS-Industrial]: http://wiki.ros.org/Industrial
+[ROS wiki]: http://wiki.ros.org/motoman
+[motoman_experimental]: https://github.com/ros-industrial/motoman_experimental
+[subversion repository]: https://code.google.com/p/swri-ros-pkg/source/browse
```

Comment:
Should this perhaps point to [ros-industrial/swri-ros-pkg](https://github.com/ros-industrial/swri-ros-pkg)?